### PR TITLE
[17.0][ADD] mrp_mass_production_order: New module mrp_mass_production_order

### PR DESCRIPTION
**Pull request by peluko00**
This module allows to create multiple manufacturing orders through a wizard. 
The wizard have the possiblity to put all manufacturing orders in state done or confirmed.

cc https://github.com/APSL 160212
@miquelalzanillas @lbarry-apsl @javierobcn @mpascuall @BernatObrador @ppyczko please review

[Grabación de pantalla desde 2024-09-09 13-53-07.webm](https://github.com/user-attachments/assets/d3834f1e-3973-4e73-a87d-97853584ad5d)

### CODE REVIEW

**Comment by mpascuall** [comment]
This error is raised when clicking on the **Bill of Material** field:
`UncaughtPromiseError > InvalidDomainError
Uncaught Promise > Invalid domain representation: &,|,company_id,=,false,company_id,=,,&,|,product_id,=,41,&,product_id.product_tmpl_id.product_variant_ids,=,41,product_id,=,false,type,=,normal`

File: mrp_mass_production_order/readme/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+1.  Go to Manufacture \> Manufacturing Orders \> Create Multiple Production Orders.
```
Comment (BernatObrador) [inline mrp_mass_production_order/readme/USAGE.md]: Maybe this point will be better explained like this:
Manufacturing > Operations > Massive MRP Production Orders

**Comment by peluko00** [comment]
> This error is raised when clicking on the **Bill of Material** field: `UncaughtPromiseError > InvalidDomainError Uncaught Promise > Invalid domain representation: &,|,company_id,=,false,company_id,=,,&,|,product_id,=,41,&,product_id.product_tmpl_id.product_variant_ids,=,41,product_id,=,false,type,=,normal`

Solved, thanks!

**Comment by mpascuall** [comment]
The Product Unit of Measure is not working correctly, it automatically set it to units once you confirm the order

**Comment by peluko00** [comment]
> The Product Unit of Measure is not working correctly, it automatically set it to units once you confirm the order

I fixed the problems with the compute's. Can you review again

**Review by mpascuall — APPROVED** [review]
Now it works correctly, tested in runboat

**Review by BernatObrador — APPROVED** [review]
LGTM, tested in runboat

**Review by ppyczko — APPROVED** [review]
LGTM, tested in runboat and works as intended

**Comment by miquelalzanillas** [comment]
Hi @peluko00 ,

During my first functional test I thought that the orders wasn't created.  

A redirection after click on the _Create_ button to a list view with the result (created orders) would be great in this module.

**Comment by peluko00** [comment]
> Hi @peluko00 ,
> 
> During my first functional test I thought that the orders wasn't created.
> 
> A redirection after click on the _Create_ button to a list view with the result (created orders) would be great in this module.

Thanks for the feedback, I added a redirection with a filter records of manufacturing orders created

**Review by miquelalzanillas — APPROVED** [review]
Functional review. LGTM

**Comment by peluko00** [comment]
Hi @pedrobaeza coud you review that when you can. Thanks!

**Comment by pedrobaeza** [comment]
Being a new module and reviewed by your team, let's merge it.

/ocabot merge nobump

**Pull request merged**